pkg/container/firecracker.go

```diff
@@ -136,6 +137,16 @@ func ExecuteFirecracker(vm *api.VM, dhcpIfaces []DHCPInterface) error {
 
 	installSignalHandlers(ctx, m)
 
+	// Create file names for persistent files to save FIFO stream to
+	logSocketPathPersistent := path.Join(vm.ObjectPath(), constants.LOG_FIFO_PERSISTENT)
+	metricsSocketPathPersistent := path.Join(vm.ObjectPath(), constants.METRICS_FIFO_PERSISTENT)
+
+	runtime.GOMAXPROCS(3)
```

Comment:
The rebase on this patch is looking healthy and passing tests.

@gpauloski, do you still remember why you added this change to GOMAXPROCS and MAX_CONCURRENT_WRITERS ?

If we could justify these things in some comments, that would be very excellent.

Thanks for your patience on this PR review.
This is a very desirable feature :)

Reply:
Apologies for the very overdue response. I do not recall why I added this change but it seems strange---maybe an artifact of being new to Go at the time.

I'm going to close this since its been a while :).